BusyList/TaskItem.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+
+namespace BusyList
+{
+    public class TaskItem
+    {
+        public TaskItem(int id, string description, TaskStatus status = TaskStatus.NotStarted)
+        {
+            Id = id;
+            Description = description;
+            TaskStatus = status;
+        }
+        public int Id { get; }
+
+        public string Description { get; }
```

Comment:
if the user makes a spelling mistake, he won't be able to change the description

Reply:
We don't have an edit command at this point anyways, so its a moot point.

Reply:
should we set this as init